.travis.yml

```diff
@@ -42,6 +42,8 @@ before_install:
 - openssl aes-256-cbc -K $encrypted_e08b3ea16ca0_key -iv $encrypted_e08b3ea16ca0_iv -in secrets.tar.enc -out secrets.tar -d
 - tar xvf secrets.tar
 - mv strings_private_app_pref.xml app/src/main/res/values/strings_private_app_pref.xml
+- curl -O https://btt-writer-resources.s3.amazonaws.com/containers.zip
```

Comment:
Can we change this to `- wget --no-verbose "https://btt-writer-resources.s3.amazonaws.com/containers.zip"` we decided to use wget in the otter build, so this would standardize.

Reply:
Yep, feel free to push the change to the cro-working branch.